### PR TITLE
Update twine-validator.txt

### PR DESCRIPTION
**Pull request by LisaGumprich**

### CODE REVIEW

File: twine-validator.txt [hunk 1]
```diff
@@ -62,197 +80,259 @@ de = Einstellungen
...
 tags = common
 [app_information_message_open_source]
- de = Die App verwendet die folgenden Open-Source-Bibliotheken:
- tags = common
+de = Die App verwendet die folgenden Open-Source-Bibliotheken:
+en = The app uses the following open-source libraries:
+tags = common
 [app_information_version_label]
 de = Version %@
+en = Version %d
```
Comment (timokoenig) [inline twine-validator.txt:322]: Should be `Version %@`

**Review by timokoenig — CHANGES_REQUESTED** [review]

File: twine-validator.txt (hunk 1 above)
Comment (LisaGumprich) [reply twine-validator.txt]: Done! :)

File: twine-validator.txt [hunk 2]
```diff
@@ -62,197 +80,259 @@ de = Einstellungen
...
 [validation_check_popup_unsuccessful_certificate_technical_problems_message]
 de = Die Signatur ist nicht gültig oder der QR-Code ist nicht lesbar.
+en = The signature is invalid or the QR code cannot be read.
 tags = common
 [validation_check_popup_unsuccessful_certificate_button_title]
 de = Weiteres Zertifikat scannen
+en = Scan another certificate
 [validation_check_popup_test_title]
 de = Negativer Schnelltest von vor %d Stunden
+en = Negative rapid test from %@ hours ago
```
Comment (timokoenig) [inline twine-validator.txt:222]: Please use format as in German translation: `%d`

File: twine-validator.txt [hunk 3]
```diff
@@ -62,197 +80,259 @@ de = Einstellungen
...
 de = Zeitpunkt der Probenahme
+en = Time sample taken
 tags = common
 [validation_check_popup_test_button_title]
 de = Weiteres Zertifikat scannen
+en = Scan another certificate
 tags = common
 [validation_check_popup_valid_pcr_test_title]
 de = Negativer PCR-Test von vor %d Stunden
+en = Negative PCR test from %@ hours ago
```
Comment (timokoenig) [inline twine-validator.txt:242]: Please use format as in German translation: `%d`

**Review by timokoenig — APPROVED** [review]

**Pull request merged**